its/projects/ProjectUnderTest/ProjectUnderTest/CSProj1.csproj

```diff
@@ -7,8 +7,8 @@
     <ProjectGuid>{1049030E-AC7A-49D0-BEDC-F414C5C7DDD8}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>ProjectUnderTest</RootNamespace>
-    <AssemblyName>ProjectUnderTest</AssemblyName>
+    <RootNamespace>CSProj1</RootNamespace>
```

Comment:
Assembly name was matching a test pattern and so `Console.WriteLine` issue was no longer raised.